data/pokemon.json

```diff
@@ -1611,7 +1611,8 @@
       "Body Slam",
       "Dazzling Gleam",
       "Disarming Voice",
-      "Play Rough"
+      "Play Rough",
+      "Dazzling Gleam"
```

Comment:
Why did this need to happen? It already exists at - https://github.com/PokemonGoF/PokemonGo-Bot/blob/dev/data/pokemon.json#L1612